packages/nextjs/components/HooksExample.tsx

```diff
@@ -122,6 +153,45 @@ const HooksExample: React.FC = () => {
             )}
           </div>
         </div>
+        {/* ETH Balance */}
+        <div className="rounded-[5px] bg-base-100 border border-gradient p-4 relative shadow">
+          <div className="trapeze"></div>
+          <h2 className="text-xl font-semibold mb-4">
+            Contract ETH Balance (useScaffoldEthBalance)
+          </h2>
+          {isEthBalanceLoading ? (
+            <p>Loading contract...</p>
+          ) : contract ? (
+            <p>{ethBalance} ETH</p>
+          ) : (
+            <p>No contract loaded</p>
+          )}
+        </div>
+
+        {/* STRK Balance */}
+        <div className="rounded-[5px] bg-base-100 border border-gradient p-4 relative shadow">
+          <div className="trapeze"></div>
+          <h2 className="text-xl font-semibold mb-4">
+            Contract STRK Balance (useScaffoldStrkBalance)
```

Comment:
Just me being a bit nitpicking but can you render all hook names with `code` style? To better distinguish the text

Reply:
done!